src/pages/index.js

```diff
@@ -2,75 +2,29 @@ import {nanoid} from 'nanoid';
 import {useState} from 'react';
 import {Helmet} from 'react-helmet';
 
-import Button from '../components/Button';
 import Layout from '../components/Layout';
-import useFetch from '../hooks/useFetch';
-import useStore from '../hooks/useStore';
+import QuestionComponent from '../components/QuestionComponent';
 
 export default function HomePage() {
-	// Data
-	const {data, loading, error} = useFetch('/api/hello');
-
-	// Local state
-	const [id, setId] = useState(null);
-
-	// Global state ZUSTAND
-	const counter = useStore(state => state.counter);
-	const decrementCounter = useStore(state => state.decrementCounter);
-	const incrementCounter = useStore(state => state.incrementCounter);
-	const setCounter = useStore(state => state.setCounter);
+	const [questionArray, setQuestionArray] = useState([
+		{questionText: 'What is export default?'},
```

Comment:
Currently you don't need to update this array and therefore, you don't need a state at this point. Saving you array in a `const` should be enough for now.